### ISSUE TITLE
Any roadmaps ?

### ISSUE THREAD

**Issue by skt041959** [veonim/veonim#650]
Hi Developer,

I can't find any community to ask questions, so I have to post it with issue. Apologize for any bothering.  Thanks for creating this interesting project. 
And do you have any further plans or road-maps to show? Since all the milestones have been reached.

Thanks

**Comment by skbolton** [comment]
Quite a bit still going on here https://github.com/veonim/veonim/blob/master/TODO.md

**Comment by breja** [comment]
Thanks for the interest!

I still have many things planned, but not always enough time. 😉

Currently in progress is work relating to polish, stability, and performance. As briefly described on the readme (https://github.com/veonim/veonim#project-status) I'm doing some exploratory research to see what it would take to find a lighter alternative to Electron (without a complete rewrite). Incidentally the current proposed approach would also allow remote editing.

Once that's figured out, it will be back to finishing up VSCode extension support for language servers, adding new neovim features like `ext_windows`/floating windows, and making sure builds are consistently stable across all platforms.

Of course there are a number of bugs and other daily editor blocker items like international input methods that should be resolved.

Once things are more stable and polished I'll be looking at feature requests and wishlist items in TODO.md as @skbolton mentioned.

I like to jump between a number of different things so I don't get bored/burned out and as such I don't make specific commitments anymore. However I do try to keep the "project status" section on the readme up to date and any other backlog items in TODO.md or issues.

tl;dr
- active stuff: https://github.com/veonim/veonim#project-status
- backlog: issues && https://github.com/veonim/veonim/blob/master/TODO.md

**Comment by skt041959** [comment]
Thank you very much for your detailed instructions.
To get rid of Electron will be a very exciting feature. I will close this issue and follow with interest of TODO.md.

Thanks.
